### PR TITLE
New package: G0rocks.Cab version V0.0.0-Alpha

### PR DESCRIPTION
**Pull request by G0rocks**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140254)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 776.)

**Comment by stephengillie** [comment]
Manual Validation ended with: 
image

File: manifests/g/G0rocks/Cab/V0.0.0-Alpha/G0rocks.Cab.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: G0rocks.Cab
+PackageVersion: V0.0.0-Alpha
+InstallerType: portable
+Installers:
```
Comment (stephengillie) [inline manifests/g/G0rocks/Cab/V0.0.0-Alpha/G0rocks.Cab.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 787.)

**Comment by G0rocks** [comment]
> Manual Validation ended with: image

How do I fix this problem?
According to this page: https://www.epicgames.com/help/en-US/c-Category_EpicGamesStore/c-EpicGamesStore_LauncherSupport/how-do-i-fix-the-vcruntime140-dll-is-missing-or-vcomp110-dll-was-not-found-error-a000084623
Then the problem is that you need to reinstall the Visual C++ redistributables on your computer.
Thanks :)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**